modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -6,22 +6,28 @@
 [id="granting-users-permission-to-monitor-user-defined-projects_{context}"]
 = Granting users permission to monitor user-defined projects
 
-Cluster administrators can monitor all core {product-title} and user-defined projects.
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
 
-Cluster administrators can grant developers and other users permission to monitor their own projects. Privileges are granted by assigning one of the following monitoring roles:
+You can also grant developers and other users different permissions:
 
-* The *monitoring-rules-view* cluster role provides read access to `PrometheusRule` custom resources for a project.
+* To monitor user-defined projects.
+* To configure the components that monitor user-defined projects.
+* To configure alert routing for user-defined projects.
 
-* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and delete `PrometheusRule` custom resources for a project. It also grants a user the ability to silence alerts.
+You can grant the permissions by assigning one of the following monitoring roles:
 
-* The *monitoring-edit* cluster role grants the same privileges as the `monitoring-rules-edit` cluster role. Additionally, it enables a user to create new scrape targets for services or pods. With this role, you can also create, modify, and delete `ServiceMonitor` and `PodMonitor` resources.
+|===
+|Cluster role name |Description
 
-You can also grant users permission to configure the components that are responsible for monitoring user-defined projects:
+|`monitoring-rules-view` | Users with this role have read access to `PrometheusRule` custom resources for a user-defined project. They can also view the *Alerts* page in the *Developer* perspective of the {product-title} web console.
 
-* The *user-workload-monitoring-config-edit* role in the `openshift-user-workload-monitoring` project enables you to edit the `user-workload-monitoring-config` `ConfigMap` object. With this role, you can edit the `ConfigMap` object to configure Prometheus, Prometheus Operator, and Thanos Ruler for user-defined workload monitoring.
+|`monitoring-rules-edit` | Users with this role can create, modify, and delete `PrometheusRule` custom resources for a user-defined project. They can also silence alerts.
```

Comment:
```suggestion
|`monitoring-rules-edit` | Users with this role can create, modify, and delete `PrometheusRule` custom resources for a user-defined project. They can create and silence alerts in the `Developer` perspective in the web console
```
see from: https://docs.openshift.com/container-platform/4.15/observability/monitoring/managing-alerts.html

Reply:
Hello @juzhao, I noticed that in the original list we did not include `monitoring-alertmanager-edit` , I could add that role to the table as well, WDYT?

Reply:
@eromanova97 monitoring-alertmanager-edit and cluster-monitoring-view,  thery are used to silence alert, this PR does not contain silence alert. @simonpasquier WDYT?

Reply:
monitoring-alertmanager-edit and cluster-monitoring-view roles are specific to platform components while this PR is about user-defined monitoring. We should be more explicit about these roles but probably not in this PR?

Reply:
Yes, I was thinking about this as well. Maybe for now I will just include the user-defined list as it was intended. We can raise a new issue to create something similar for other roles. :+1: thank you for your inputs!